.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
   integration_test:
     machine:
-      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
```

Comment:
What image will this use?

Reply:
It uses the default circle machine image which is what we use on our other tests, all we expect it to provide is docker.  Is there something special we need from that particular image?

Reply:
The default image is really old, but we can watch and see if it causes any problems.